docs/release_notes.py

```diff
@@ -0,0 +1,93 @@
+import sys
```

Comment:
Should upstream this to `pcdsutils` one day (but not today)

Reply:
Yes